### PR TITLE
Closes #495 Cyril/show member info

### LINKED ISSUE #495: Add member information for Projects
After selecting a project and seeing all samples in the project the user should see three things in the right-hand corner: 
1. If the project is private (hidden) or public (visible)
2. How many members are part of a project
3. A button to add project members

### PR DESCRIPTION
**Pull request by cyrielo**
# Description
This PR adds a project menu list to the subheader to view total members in a project, a button to add a user to the project and project visibility.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Impacted Areas in Application
Which pages?

- [x] All Projects Page

Specific components 

* Samples

# Testing Script:

### All Project Page
* Switch between projects 
* Search for a sample
* Navigate through different pages of samples
* Select Upload new sample button
* Select sample to see generated report

# Checklist:

- [x] I have run through the testing script to make sure current functionality is unchanged
- [x] I have done relevant tests that prove my fix is effective or that my feature works
- [x] I have spent time testing out edge cases for my feature
- [x] I have updated the test script or pull request template if necessary
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/controllers/projects_controller.rb [hunk 1]
```diff
@@ -15,6 +15,20 @@ def index
   # GET /projects/1
   # GET /projects/1.json
   def show
+    # all exisiting project are null, we ensure private projects are explicitly set to 0
+    public_access = @project.public_access.nil? ? 1 : @project.public_access
```
Comment (cdebourcy) [inline app/controllers/projects_controller.rb]: I think it's a better default to set the variable to 0 when it is nil

Comment (cyrielo) [reply app/controllers/projects_controller.rb]: I assume 0 means private right? so that means all projects are private by default?

Comment (cdebourcy) [reply app/controllers/projects_controller.rb]: Yes. Until we set them to public explicitly

Comment (cyrielo) [reply app/controllers/projects_controller.rb]: Okay, updating ...

File: app/assets/src/components/Samples.jsx [hunk 2]
```diff
@@ -740,22 +736,61 @@ class Samples extends React.Component {
...
+               <span>
+                 <i className="tiny material-icons">lock</i> Private Project
+               </span>
+            ) : null }
+          </li>
+          <li>
+              { this.state.project ? (
+                this.state.project.total_members ?
+                <span>
+                  <i className="tiny material-icons">people</i> {this.state.project.total_members} Member(s)
```
Comment (cdebourcy) [inline app/assets/src/components/Samples.jsx]: In this PR or the next, please get rid of the "(s)" and check the plural to make it correct. There must be a "pluralize" function out there that does it automatically

**Pull request merged**